### DISCUSSION TITLE
Shared /data across multiple instances of containers (with different worlds)

### DISCUSSION THREAD

**Discussion by ajma** [itzg/docker-minecraft-bedrock-server#310] [Q&A]
I have multiple docker containers setup (one for each of my kids) and they're currently all setup to individually have their own /data folders. Could I point all the docker containers to the same /data folder and set the LEVEL_NAME environment var to different worlds?

**Comment by itzg** [comment]
That might work, since the open files during runtime will be located within each level/world directory. Backup the data and give it a try. 

I am wondering though what advantage that will provide?

**Reply by ajma** [reply]
The data volume I'm using is a mounted NFS share that lives on my NAS. The container is running on a cheap micro desktop. I'm just trying to make it easier for me to create new worlds when my kids want another one if I can just have a single NFS mount and just point the container to another world.
